include/bluetooth/audio.h

```diff
@@ -36,6 +36,21 @@ struct bt_audio_cap;
 
 #define BT_BROADCAST_ID_SIZE             3 /* octets */
 
+/* Audio Context Type, Generic Audio */
+#define BT_AUDIO_CONTEXT_TYPE_PROHIBITED         0x0000
+#define BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED        0x0001
+#define BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL     0x0002
+#define BT_AUDIO_CONTEXT_TYPE_MEDIA              0x0004
+#define BT_AUDIO_CONTEXT_TYPE_GAME               0x0008
+#define BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL      0x0010
+#define BT_AUDIO_CONTEXT_TYPE_VOICE_ASSISTANTS   0x0020
+#define BT_AUDIO_CONTEXT_TYPE_LIVE               0x0040
+#define BT_AUDIO_CONTEXT_TYPE_SOUND_EFFECTS      0x0080
+#define BT_AUDIO_CONTEXT_TYPE_NOTIFICATIONS      0x0100
+#define BT_AUDIO_CONTEXT_TYPE_RINGTONE           0x0200
+#define BT_AUDIO_CONTEXT_TYPE_ALERTS             0x0400
+#define BT_AUDIO_CONTEXT_TYPE_EMERGENCY_ALARM    0x0800
```

Comment:
You can use the `BIT(x)` macro to make this a bit easier to read, e.g.

```
#define BT_AUDIO_CONTEXT_TYPE_PROHIBITED         0
#define BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED        BIT(1)
#define BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL     BIT(2)
```
etc.

Reply:
> You can use the `BIT(x)` macro to make this a bit easier to read, e.g.
> 
> ```
> #define BT_AUDIO_CONTEXT_TYPE_PROHIBITED         0
> #define BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED        BIT(1)
> #define BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL     BIT(2)
> ```
> 
> etc.

Great point, will do it.